djangocms_navigation/helpers.py

```diff
@@ -0,0 +1,21 @@
+from django.contrib.contenttypes.models import ContentType
+
+from .models import MenuItem
+
+
+def get_content_object_from_navigation_tree(menu_content, content_object):
```

Comment:
Sorry mate, I hate to be a pain. The name of the helper is very important as it will be used a lot in the future and documented. 

1. A better name that describes what it is doing is:

`get_navigation_node_for_content_object`

2. There needs to be an additional argument with a default as the package is reused in FIL: node_model=MenuItem

3. The docstring should describe the in and out the same as the other packages, versioning is the best example of this: https://github.com/divio/djangocms-versioning/blob/master/djangocms_versioning/helpers.py#L21:

    """
    Find a navigation node that contains a content_object in a Navigation menu

    :param menu_content: A MenuContent instance
    :param content_object:  A content object registered in the cms_config
    :param node_model:  A model used for a navigation item
    :return: A navigation node or False
    """